src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -364,8 +364,11 @@ def identify_subtrees_and_overlaps(self,
                     if prev_root_id != root_id:
                         overlapped_roots.add(prev_root_id)
                         overlapped_roots.add(root_id)
-                        self.log.error('pool %d has overlapping roots: %s',
-                                       pool_id, overlapped_roots)
+                        self.log.warning('pool %s has overlapping roots: %s',
+                                       pool['pool_name'], overlapped_roots)
```

Comment:
let's inform the user what this implies - add something like "won't scale"?

Reply:
agreed